work.html

```diff
@@ -64,10 +56,48 @@ <h2>We're not currently hiring, but...</h2>
 
 </section>
 
+
 <section>
-  <h2>Previous roles</h2>
+  <a name="mini-apprenticeship"></a>
+  <h3>Mini apprenticeship (6 weeks)</h3>
+
+  <p>Position Dev is piloting a mini apprenticeship program! We’re looking for a junior developer to join us for 6 weeks
+  on-site at our New York City office to work on the Wayland magazine platform, which is used by Jacobin, Current
+  Affairs,and Monthly Review.</p>
+
+  <h4>What you’ll be doing</h4>
+
+  <p>You’ll create and style web pages with HTML, CSS, and JavaScript. You’ll learn how to build a web app with Haskell,
+  and  get the chance to contribute to open-source Haskell libraries. You’ll solve real-world problems that affect users, and
+  work with experienced developers to build features that bring value to our clients.</p>
+
+  <h4>We’re looking for someone who can:</h4>
+  <ul>
+    <li>Create and style web pages with HTML and CSS</li>
+    <li>Use Vue, jQuery, and vanilla JavaScript to add interactivity to web pages</li>
+    <li>Create forms that communicate with a backend</li>
+    <li>Build the backend for a form</li>
+    <li>Collaborate with other developers</li>
+    <li>Make commits and branches and push to GitHub</li>
+    <li>Use the command line to navigate a file system and run programs</li>
+  </ul>
```

Comment:
Maybe add a note here to explicitly say that backend skills can be using any programming language and no previous knowledge of Haskell is neccesary.